cloudferrylib/os/network/neutron.py

```diff
@@ -881,6 +894,14 @@ def upload_networks(self, networks):
             LOG.debug("Trying to create network '%s'", net['name'])
             tenant_id = identity.get_tenant_id_by_name(net['tenant_name'])
 
+            if tenant_id is None:
```

Comment:
Maybe it is better to use `try_get_tenant...` method from identity resource here?